masterlist.yaml

```
@@ -2084,6 +2084,15 @@ plugins:
       - *alreadyInOrFixedByFCF
       - *alreadyInOrFixedByPRPv0_59
 
+  - name: 'Better Landscape Grass.esp'
+    url: ["https://www.nexusmods.com/fallout4/mods/38360/" ]
```

The masterlists use single-quotes to encapsulate strings. You can also add a name to display the link with in the UI:

```suggestion
    url:
      - link: 'https://www.nexusmods.com/fallout4/mods/38360/'
        name: 'Better Landscape Grass'
```